cob_obstacle_distance_moveit/launch/obstacle_distance.launch

```diff
@@ -17,6 +17,7 @@
     <arg name="use_sensors" value="$(arg use_sensors)"/>
   </include>
 
-  <node pkg="cob_obstacle_distance_moveit" type="obstacle_distance_node" name="$(arg node_name)" output="screen"/>
-
+  <node pkg="cob_obstacle_distance_moveit" type="obstacle_distance_node" name="$(arg node_name)" output="screen">
+    <!--remap from="/obstacle_distances" to="/obstacle_distances"/-->  <!--only needed for the cob4 specific configuration-->
```

Comment:
not sure, it has an extra "/" I just copied from https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/tools/obstacle_distance_monitor.launch#L26 because I opened a pull request to remove the file https://github.com/ipa320/cob_robots/pull/478/files and I wanted to save the remap information.